tests/unit_node/sqlite_test.ts

```diff
@@ -174,7 +174,6 @@ Deno.test("[node/sqlite] applyChangeset across databases", () => {
   const changeset = session.changeset();
   targetDb.applyChangeset(changeset, {
     filter: (e) => e === "data",
-    // @ts-ignore: types are not up to date
```

Comment:
You might need to bump the local version of `@types/node` in tests/registries/npm